client/ios/transports/gemini-websocket.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: "Gemini Live Websocket Transport"
+description: "Websocket implementation for iOS using Gemini"
+---
+
+The Gemini Live Websocket transport implementation enables real-time audio communication with the Gemini Multimodal Live service, using a direct websocket connection.
+
+<Note>
+  Transports of this type are design primarily for development and testing
```

Comment:
```suggestion
  Transports of this type are designed primarily for development and testing
```

Reply:
https://github.com/pipecat-ai/docs/pull/169

Reply:
Yep. 😅
Thank you, fixed!